src/EventListener/AssetsInstallCommandListener.php

```diff
@@ -75,7 +75,7 @@ public function copySonataCoreBundleAssets(ConsoleTerminateEvent $event)
         $this->execute($event->getInput(), $event->getOutput(), $application);
     }
 
-    protected function execute(InputInterface $input, OutputInterface $output, FrameworkApplication $application): int
+    private function execute(InputInterface $input, OutputInterface $output, FrameworkApplication $application): int
```

Comment:
This seems weird

Reply:
can you report an issue on php-cs-fixer please?